core/commands/config.go

```diff
@@ -41,15 +41,15 @@ repository.`,
 much like 'git config'. The configuration values are stored in a config
 file inside your IPFS repository.
 
-EXAMPLES:
+Examples:
 
 Get the value of the 'datastore.path' key:
 
-  ipfs config datastore.path
+  $ ipfs config datastore.path
```

Comment:
I think not using them in man pages is smart, but using them in READMEs makes sense. 

cf https://github.com/ipfs/community/pull/106